tardis/plasma/properties/nlte_rate_equation_solver.py

```diff
@@ -961,7 +959,7 @@ def recomb_matrix(recomb_coefficients, atomic_number, ion_number):
     offdiag = np.zeros(atomic_number)
     index = recomb_coefficients.index
     for i in index:
-        offdiag[i] = recomb_coefficients.loc[i]
+        offdiag[i] = float(recomb_coefficients.loc[i].values[0])
```

Comment:
Why does this use `.loc.values`?

Reply:
it does the same thing as iloc, I will update the previous function with values instead of iloc